File: app/src/main/java/com/example/dcct/view/RecordCallback.java
package com.example.dcct.view;

import com.example.dcct.base.BaseCallback;
import com.example.dcct.bean.BackResultData;
import com.example.dcct.bean.Record;

import java.util.List;

public interface RecordCallback extends BaseCallback {
    void onLoadQueryData(BackResultData<List<Record>> backResultData);
    void onLoadError(Throwable t);
}

File: app/src/main/java/com/example/dcct/view/RecordCallback.kt
package com.example.dcct.view

import com.example.dcct.base.BaseCallback
import com.example.dcct.bean.BackResultData
import com.example.dcct.bean.Record

interface RecordCallback : BaseCallback {
    fun onLoadQueryData(backResultData: BackResultData<List<Record>>)
    fun onLoadError(t: Throwable)
}